pytest_plugins/markers.py

```diff
@@ -10,6 +10,7 @@ def pytest_configure(config):
         "tier2: Tier 2 tests",  # Association tests
         "tier3: Tier 3 tests",  # Systems integration tests
         "tier4: Tier 4 tests",  # Long running tests
+        "tier5: Tier 5 tests",  # Deprecated component tests
```

Comment:
I think I missed a discussion around this new tier designation. What is the reasoning behind this?

Reply:
yes, this new tier is for deprecated components, which will be low prio